### PR TITLE
test the home page route

### PR DESCRIPTION
**Pull request by Suuuuha**

### CODE REVIEW

**Review by mattlub — APPROVED** [review]

File: app/test/test.js [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
 tape('basic passing test, to check tape is functioning correctly', t => {
   t.ok(true, 'meaningless test should pass')
   t.end()
 })
+
+tape('home route test', t => {
+  request(app)
+    .get('/')
+    .end(function (req, res) {
+      // console.log(res)
```
Comment (Karyum) [inline app/test/test.js]: do we need this console log here ?

File: app/test/test.js [hunk 2]
```diff
@@ -1,6 +1,18 @@
 const tape = require('tape')
+const request = require('supertest')
+const app = require('../src/server.js')
 
 tape('basic passing test, to check tape is functioning correctly', t => {
```
Comment (Karyum) [inline app/test/test.js]: do you still need this test?

**Review by Karyum — CHANGES_REQUESTED** [review]
some small changes

**Review by mattlub — APPROVED** [review]

**Review by Karyum — APPROVED** [review]
FILEM

**Pull request merged**